### PR TITLE
Fix user status icon in avatar menu

### PR DESCRIPTION
**Pull request by nickvergessen**
Before | After
---|---
Bildschirmfoto vom 2023-01-04 10-18-55 | Bildschirmfoto vom 2023-01-04 10-17-55

## Summary

The user status icon was also color inverted on dark mode

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [x] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [x] Screenshots before/after for front-end changes
- [x] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [x] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

File: core/css/apps.scss [hunk 1]
```diff
@@ -973,7 +973,13 @@ $outter-margin: math.div($popoveritem-height - $popovericon-size, 2);
 .contact .popovermenu ul,
 .popover__menu {
 	> li > a > img {
-		 filter: var(--background-invert-if-dark);
+		filter: var(--background-invert-if-dark);
+
+		// We do not want to invert the color of the user_status emoji:
+		// https://github.com/nextcloud/nextcloud-vue/blob/8899087f8f8d45e0ed744bde9faa00b625a21905/src/components/NcAvatar/NcAvatar.vue#L495
+		&[src^="data"] {
+			filter: none !important;
```
Comment (szaimen) [inline core/css/apps.scss:981]: are you sure the important is needed here?

Comment (nickvergessen) [reply core/css/apps.scss:981]: Indeed, works without

**Review by szaimen — APPROVED** [review]
LGTM but didnt test

**Review by skjnldsv — APPROVED** [review]

**Pull request merged**